### ISSUE TITLE
Upgrade facebook sdk to 5.0

### ISSUE THREAD

**Issue by happyharis** [roughike/flutter_facebook_login#133]
Please upgrade facebook login sdk version.

Now: com.facebook.android:facebook-login:4.39.0
Now: ios :SDK v4.41.0 and above 

Latest: com.facebook.android:facebook-android-sdk:
Latest iOS: SDK v5.0 and above

**Comment by hientranea** [comment]
I have the same issue. Facebook gives us the warning about upgrading our app SDK to v5.0 for IOS platform. But the latest version `flutter_facebook_login` ( v 2.0.1) specify :
> s.dependency 'FBSDKCoreKit', '4.39.1'
>  s.dependency 'FBSDKLoginKit', '4.39.1'

Please consider upgrade FB Login SDK version.

**Comment by Pebsie** [comment]
I've just received an email threatening deletion of the app if this isn't updated. This plugin doesn't work until this is resolved.

**Comment by feinstein** [comment]
FYI 5.2.0 is the newest version

**Comment by ahasnaini** [comment]
Any update on this, having the same issue

**Comment by vijayawoshe** [comment]
I have the same issue. please do needfull

**Comment by vijayawoshe** [comment]
we have the same issue. please do needful

**Comment by monmonja** [comment]
incase anyone wants to use other version more updated within the network https://github.com/roughike/flutter_facebook_login/network you can update your pubspec to something like
```dart
flutter_facebook_login:    
    git: https://github.com/ragnor-rs/flutter_facebook_login    
    version: 2.0.1
```

**Comment by rezashahbazi** [comment]
Actually I upgraded `FBSDKCoreKit` and `FBSDKLoginKit` to '5.5.0'

```
flutter_facebook_login:
    git: https://github.com/rezashahbazi/flutter_facebook_login
    version: 2.0.2
```

Make sure to remove Podfile.lock and run `pod install`

**Comment by roughike** [comment]
Fixed in `3.0.0`.
